pallets/runtime/develop/src/runtime.rs

```diff
@@ -95,9 +103,14 @@ pub const VERSION: RuntimeVersion = RuntimeVersion {
     spec_name: create_runtime_str!("polymesh"),
     impl_name: create_runtime_str!("polymath-polymesh"),
     authoring_version: 1,
+    // Per convention: if the runtime behavior changes, increment spec_version
+    // and set impl_version to 0. If only runtime
+    // implementation changes and behavior does not, then leave spec_version as
+    // is and increment impl_version.
     spec_version: 1004,
     impl_version: 1004,
```

Comment:
Let's reset these versions as per the comment above?

Reply:
Yeah, from what base no. you want to start. 1000? or 0. Although this PR changes come under the implementation changes & behaviour changes as well.

Reply:
Or keep same base no. and changed it to 1005 to both.?

Reply:
Aldebaran is on 1007 at this point. Would prefer a different "namespace", maybe `2000`?

Reply:
2000, 0 sounds good to me